### PR TITLE
Cb 91

### PR DESCRIPTION
**Pull request by CalebMacdonaldBlack**

### CODE REVIEW

File: src/cljs/centrebull/aggregates/views.cljs [hunk 1]
```diff
@@ -73,7 +73,7 @@
    [aggregate-section aggregates remove]
    [:section
     [:card
-     [:h4 "Create new Activity"]
+     [:h4 "Create new aggregate"]
```
Comment (axrs) [inline src/cljs/centrebull/aggregates/views.cljs]: Capital A

File: src/cljs/centrebull/grand-aggregates/core.cljs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+(def pages
+  {:grand-aggregates #'page
+   :grand-aggregate-page #'single})
+
+(secretary/defroute "/grand-aggregates/:id" {id :id :as params}
+  (rf/dispatch [:set-active-grand-aggregate id]))
+
+(secretary/defroute "/grand-aggregates" []
+  (rf/dispatch [:grand-aggregates-load]))
```
Comment (axrs) [inline src/cljs/centrebull/grand-aggregates/core.cljs]: New Line

File: src/cljs/centrebull/grand-aggregates/handlers.cljs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      (if competition-id
+        (get-json {:url           (str "competitions/" competition-id "/grand-aggregates")
+                   :after-success [[::set-active-grand-aggregates]]})
+        {}))))
+
+(reg-event-fx
+  :grand-aggregates-load
+  (fn [_ _]
+    {:dispatch-n [[:set-active-page :grand-aggregates]
+                  ; [:refresh-activities]
```
Comment (axrs) [inline src/cljs/centrebull/grand-aggregates/handlers.cljs]: Remove comment.

File: src/cljs/centrebull/grand-aggregates/handlers.cljs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  (fn [{:keys [db]} [_ id]]
+    {:db         (assoc db :active-grand-aggregate (first (filter #(= id (:grand-aggregate/id %)) (:grand-aggregates db))))
+     :dispatch-n [[:set-active-page :grand-aggregate-page] [:refresh-grand-aggregate-results]]}))
+
+(reg-event-fx
+  :grand-aggregates-delete
+  (fn [{:keys [db]} [_ id & after-success]]
+    (let [competition-id (get-in db [:active-competition :competition/id])]
+      (delete-json {:url           (str "competitions/" competition-id "/grand-aggregates/" id)
+                    :after-success after-success}))))
```
Comment (axrs) [inline src/cljs/centrebull/grand-aggregates/handlers.cljs]: New Line

File: src/cljs/centrebull/grand-aggregates/views.cljs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+          [:h3 "Aggregate Results"]]
+        [:card
+          [:table
+            [:thead
+              [:tr]
+              [:th "Grade"]
+              [:th "Name"]
+              [:th "Club"]
+              (for [r (:aggregate/results f)]
+                [:th "#" (:aggregate/priority r)])
```
Comment (axrs) [inline src/cljs/centrebull/grand-aggregates/views.cljs]: ^{:key}

**Review by axrs — CHANGES_REQUESTED** [review]

**Comment by CalebMacdonaldBlack** [comment]
@axrs

**Review by axrs — APPROVED** [review]

**Pull request merged**